### PR TITLE
Random State

### PR DESCRIPTION
**Pull request by Engineer-C**
In [**sklearn.model_selection.train_test_split**](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html#sklearn.model_selection.train_test_split), parameter 'random_state' can be passed.
It is important to set a random state to **make sure the result is reproducible**.

The seed of randomness '*42*' does not contain any specific meaning,
but derived from the '***Hitchhiker's Guide to the Galaxy***', by Douglas Adams.
According to the literature, it is considered as
'the Answer to the Ultimate Question of Life, the Universe and Everything'  by the Deep Thought.

### CODE REVIEW

**Comment by JEONGH0SHIN** [comment]
that is so important I think !  
from now, I will set a random state to get good results. **Thank you! Good pull request!! **
